src/Controllers/CopyPatrol.php

```diff
@@ -208,11 +208,16 @@ public function getReportLink( $ithenticateId ) {
 
 	/**
 	 * @param $datetime string Datetime of edit
+	 * @param $gmt boolean whether to convert to GMT
 	 * @return string Reformatted date
 	 */
-	public function formatTimestamp( $datetime ) {
+	public function formatTimestamp( $datetime, $gmt = false ) {
 		$datetime = strtotime( $datetime );
-		return date( 'Y-m-d h:m', $datetime );
+		if ( $gmt ) {
+			return gmdate( 'Y-m-d H:i', $datetime );
+		} else {
+			return date( 'Y-m-d H:i', $datetime );
```

Comment:
These two invocations will be the same from Tool Labs where the local server timezone is UTC. It really might be easier/better to change https://github.com/Niharika29/PlagiabotWeb/blob/master/public_html/index.php#L33-L38 so that the app always sets the timezone to UTC.